### DISCUSSION TITLE
[Feature] Customize the destination album of Backup folders

### DISCUSSION THREAD

**Discussion by Veuchez** [immich-app/immich#14622] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I haven't found a way to choose the destination album of Android backup folders. 
Now you can just Load the "Camera" folder into the "Camera" album. 
But what if I wanted to backup the "Camera" folder in the "Backup Camera Roberto" album? is it possible? 
I think this feature would be convenient for many. 
Thank you.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#12748
